sdk.js

```diff
@@ -523,6 +523,30 @@ class PaddleSDK {
 		return this._request('/subscription/users', { body });
 	}
 
+	/**
+	 * Change the due date of an upcoming subscription payment
+	 *
+	 * @method
+	 * @param {number} paymentID
+	 * @param {string} date
```

Comment:
Let's send the `date` as a real `Date` object instead.

Reply:
We could certainly, I'm not sure that exposing a different interface than the underlying API is a good idea in this case. 
One could be mislead into believing that the time portion plays a role as well.

If you're convinced, then I'll update the method signature.

Reply:
Date it is. Would you rather have an embedded left pad function for the zero fills or use just `.toISOString().substring(0, 10)` for formatting?

Reply:
Ready to be reviewed.